lib/motions/open-file-under-cursor.coffee

```diff
@@ -0,0 +1,58 @@
+{Motion} = require './general-motions'
+path = require 'path'
+{Directory} = require 'atom'
+shell = require 'shell'
+{exec} = require 'child_process'
+utils = require '../utils'
+
+class OpenFileUnderCursor extends Motion
+  execute: (count) ->
+    ###
+      /usr/bin/test
+      ../../.travis.yml
+      ./search-motion
+      ./index
+      http://github.com
+      https://thing.com/item%20test
+      //github.com
+    ###
+    wordRegex = /[a-z0-9\.\-_\/\\%:]+/i
+    @editor.getCursors().forEach (cursor) =>
+      currentWord = utils.getCurrentWord @editor, cursor, wordRegex
+      # range = cursor.getCurrentWordBufferRange wordRegex: wordRegex
+      # selectedPath = @editor.getTextInRange range
+      range = currentWord.range
+      selectedPath = currentWord.word
+
+      # exit early for url match
+      if /^https?:/i.test(selectedPath)
+        return @openUrl selectedPath
+      if /^\/\//i.test(selectedPath)
+        return @openUrl "http:#{selectedPath}"
+
+      if selectedPath[0] isnt '/' and selectedPath[0] isnt '\\'
+        selectedPath = path.join path.dirname(@editor.buffer.file.path), selectedPath
+
```

Comment:
Thanks! Not sure if that would really be useful yet, but it might be good if there is a need to extend from this class. Are there any other vim commands that work via files?